packages/pangea-sdk/.sdk-ci.yml

```
@@ -26,7 +26,7 @@ sdk-integration-tests:
     SERVICE_FILE_INTEL_ENV: LVE
     SERVICE_USER_INTEL_ENV: LVE
     SERVICE_REDACT_ENV: LVE
-    SERVICE_VAULT_ENV: LVE
+    SERVICE_VAULT_ENV: DEV
```

Undo this when Vault v2 is in LVE.
```suggestion
    SERVICE_VAULT_ENV: LVE
```